src/java.base/share/classes/sun/security/x509/SubjectAlternativeNameExtension.java

```diff
@@ -84,27 +84,20 @@ public SubjectAlternativeNameExtension(GeneralNames names)
      * criticality and GeneralNames.
      *
      * @param critical true if the extension is to be treated as critical.
-     * @param names the GeneralNames for the subject.
+     * @param names the GeneralNames for the subject, cannot be null or empty.
      * @exception IOException on error.
      */
     public SubjectAlternativeNameExtension(Boolean critical, GeneralNames names)
-    throws IOException {
+            throws IOException {
```

Comment:
`GeneralNames` should probably also be modified so that it cannot contain an empty or null List.